ote_cli/ote_cli/tools/find.py

```diff
@@ -31,6 +31,7 @@ def parse_args():
         "--root", help="A root dir where templates should be searched.", default="."
     )
     parser.add_argument("--task_type")
+    parser.add_argument("--experimental", action="store_true")
```

Comment:
maybe, but in this case I would set it to False while checking ote_cli features like train, eval, export and etc. Otherwise testing will take much more longer. However experimental templates have to be tested as well, I don't know how it can be done during reasonable time.

Reply:
This is testing code, this is a user-facing API. And IMO having this feature disabled here means that we don't want users to use "experimental" templates in OTE. So, I think about switching the default.

This being said, I agree that non-experimental templates should be tested in the first place, and experimental ones may be skipped to decrease CI load.